src/bridge/jsHelpers.js

```diff
@@ -385,30 +386,35 @@ export function makeAccountBridgeReceive({
     verify?: boolean,
     deviceId: string,
     subAccountId?: string,
-    freshAddressIndex?: string,
+    freshAddressIndex?: number,
   }
 ) => Observable<{
   address: string,
   path: string,
 }> {
-  const getFreshAddressByIndex = (account, index) => {
-    return account.freshAddresses[Number(index)];
-  };
   return (account, { verify, deviceId, freshAddressIndex }) => {
+    if (freshAddressIndex && !account.freshAddresses[freshAddressIndex]) {
```

Comment:
freshAddressIndex=0 is legit no?

maybe it should be `freshAddressIndex !== undefined`